pallets/parachain-staking/src/lib.rs

```diff
@@ -111,19 +114,23 @@ pub mod pallet {
         <<T as Config>::Currency as Currency<<T as frame_system::Config>::AccountId>>::Balance;
 
     pub const COLLATOR_LOCK_ID: LockIdentifier = *b"stkngcol";
-    pub const NOMINATOR_LOCK_ID: LockIdentifier = *b"stkngdel";
+    pub const NOMINATOR_LOCK_ID: LockIdentifier = *b"stkngnom";
```

Comment:
This is a unrelated, but a valid change. Its a simple rename in line with our previous renames.